html/ajax_search.php

```diff
@@ -1,362 +0,0 @@
-<?php
-
-use LibreNMS\Util\Debug;
-
-$init_modules = ['web', 'auth'];
-require realpath(__DIR__ . '/..') . '/includes/init.php';
-
-if (! Auth::check()) {
-    exit('Unauthorized');
-}
-
-Debug::set($_REQUEST['debug']);
-
-$device = [];
-$ports = [];
-$bgp = [];
-$limit = (int) \LibreNMS\Config::get('webui.global_search_result_limit');
-
-if (isset($_REQUEST['search'])) {
-    $search = $_REQUEST['search'];
-    header('Content-type: application/json');
-    if (strlen($search) > 0) {
-        $found = 0;
-
-        if (! Auth::user()->hasGlobalRead()) {
-            $device_ids = Permissions::devicesForUser()->toArray() ?: [0];
-            $perms_sql = '`D`.`device_id` IN ' . dbGenPlaceholders(count($device_ids)) . ' AND ';
-        } else {
-            $device_ids = [];
-            $perms_sql = '';
-        }
-
-        if ($_REQUEST['type'] == 'group') {
```

Comment:
So group and the other friends are not used anywhere?

Reply:
I could not find any other references to _search.php or ajax_search.php, So, yeah, no idea why that code was there.

Reply:
cant find any either